llama_stack/apis/post_training/post_training.py

```diff
@@ -30,18 +30,22 @@ class OptimizerConfig(BaseModel):
     lr: float
     lr_min: float
     weight_decay: float
+    num_warmup_steps: int
 
 
 @json_schema_type
 class TrainingConfig(BaseModel):
+    dtype: str
```

Comment:
no default for this? shouldn't the default be derived from the base model you are training?

Reply:
just refined the api part in the latest commit